core/domain/exp_services.py

```diff
@@ -366,6 +366,14 @@ def get_non_private_exploration_summaries():
         exp_models.ExpSummaryModel.get_non_private())
 
 
+def get_featured_exploration_summaries():
+    """Returns a dict with all featured exploration summary domain objects,
+    keyed by their id.
+    """
+    return _get_exploration_summary_dicts_from_models(
```

Comment:
In either this commit or a separate commit, could you please rename this to _get_exploration_summaries_from_models()? The inconsistency is pretty bad.

Probably best to do it in a separate commit to develop, actually, and then we can later merge that into new-gallery.